composer.json

```diff
@@ -79,6 +79,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "^3.155",
+        "composer/composer": "^2.1",
```

Comment:
should be

```
"composer-plugin-api": "^1.0|^2.0"
```

and really should be in suggest block too.

Reply:
we also use the plugin api when we do the service provider discovery... so maybe we should make this an actual requirement in laravel 9?

Reply:
Done. I don't think it should be in the suggest block because everyone has Composer installed anyway.

Reply:
having the compiler binary on ur machine is not the same as having the composer library installed :)